datalad/core/local/create.py

```diff
@@ -105,6 +105,28 @@ class Create(Interface):
         EnsureKeyChoice('action', ('create',)) & \
         EnsureKeyChoice('status', ('ok', 'notneeded'))
 
+    _examples_ = [
+        dict(text="""Create a dataset 'mydataset' in the current directory""",
+             code_py="create(path='mydataset')",
+             code_cmd="datalad create mydataset",
+             setup_py="python -c 'from datalad.api import create, remove'",
```

Comment:
It doesn't look like these `setup_` or `teardown_` snippets are used.  Until something like that is implemented, I think it'd be best to remove them.

Reply:
Gone